cnf-certification-test/identifiers/identifiers.go

```diff
@@ -209,6 +210,16 @@ The label value is not important, only its presence.`,
 		bestPracticeDocV1dot4URL+" Section 4.6.24",
 		TagExtended)
 
+	TestStorageRequiredPods = AddCatalogEntry(
+		"storage-required-pods",
+		common.LifecycleTestKey,
+		`Checks that pods not place persistent volumes on local storage.`,
```

Comment:
Should be 'pods do not place'. And why is there a mix of back quote and double quote marks? I'd use back quote everywhere:
 https://golangbyexample.com/double-single-back-quotes-go/